packages/webapp/src/components/CropCatalogue/CropStatusInfoBox/styles.module.scss

```diff
@@ -0,0 +1,23 @@
+.container {
+  display: flex;
+  padding: 12px;
+  position: relative;
+  flex-direction: column;
+  row-gap: 16px;
+}
+
+.secondRowContainer {
+  display: flex;
+  gap: 10px;
+}
+
+.cropCountContainer {
+  display: flex;
+  gap: 4px;
+  align-items: center;
+  overflow-wrap: anywhere;
```

Comment:
I considered just doing the last element of its kind -- but it looks particularly bad for Malayalam.

Reply:
With this property, double digit numbers are illegible for me at a certain breakpoint

<img width="245" alt="Screenshot 2024-09-12 at 1 35 25 PM" src="https://github.com/user-attachments/assets/d86a4ba4-fd49-41ae-8412-7144686dfc2f">

The words breaking at any point are also hard to read IMO -- may I suggest that we remove this property and instead add a `flex-wrap: wrap` on the container (`secondRowContainer`)? 

<img width="245" alt="Screenshot 2024-09-12 at 1 38 08 PM" src="https://github.com/user-attachments/assets/9a2cd064-a50a-4bd7-986c-ba276ab5e47b">

I think that it'd look better UX wise and be more legible. Thoughts?

Reply:
Woah thank you for spotting that @antsgar!! It has to be the second one because the way those letters got chunked up in Punjabi forms words of its own which is utterly confusing to read

Reply:
yup way better!